packages/common/src/ui/components/modals/ListSearch/ListSearch.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { AutocompleteList } from '../..';
+import { useWindowDimensions } from '../../../..';
+import { LocaleKey } from '../../../../intl/intlHelpers';
+import { BasicModal } from '../BasicModal';
+import { ModalTitle } from '../ModalTitle';
+
+interface ListSearchProps<T> {
+  title: LocaleKey;
+  options: T[];
+  optionKey: keyof T;
+  open: boolean;
+  onClose: () => void;
+}
+
+export const ListSearch = <T extends unknown>({
+  title,
+  options,
+  optionKey,
+  open,
+  onClose,
+}: ListSearchProps<T>): JSX.Element => {
+  const { height } = useWindowDimensions();
```

Comment:
I'm not sure if this is a very good way to do this. I think maybe trying to flex generally always ends up being a better idea